README.md

```diff
@@ -299,6 +299,71 @@ If the device where the app is running has a Custom Tabs compatible Browser, a C
 ```
 
 
+### Clearing the session
+
+To log the user out and clear the SSO cookies that the Auth0 Server keeps attached to your browser app, you need to call the [logout endpoint](https://auth0.com/docs/api/authentication?#logout). This can be done is a similar fashion to how you authenticated before: using the `WebAuthProvider` class.
+
+Note that the Manifest Placeholders must be set in order for the redirection to work, just like you were required to do so for the authentication flow. Make sure to [revisit that section](#authentication-with-universal-login) first if you still cannot authenticate successfully.
```

Comment:
What this is trying to say is that the Manifest Placeholders that the user was told to set up for authentication, are required to be present for this to work. The intent of this paragraph is to be a reminder for the user to set up that value. They will definitely do it because:
- gradle won't compile their app if they are using this SDK and not setting them up in the build.gradle file.
- probably they are already using this for Authentication. Logout is just an addition!

This is where I link them, the first time they are mentioned in the readme https://github.com/auth0/Auth0.Android#authentication-with-universal-login. Do you think this paragraph should be written in a different way @joshcanhelp ?

Reply:
Nope, this is fine with me, thank you for the explanation.